### PR TITLE
Update SetPedConfigFlag.md

### PR DESCRIPTION
**Pull request by alberttheprince**
Added the most up-to-date list of enum names to help use native documentation when setting flags.

Source:
https://alexguirre.github.io/rage-parser-dumps/dump.html?build=3095&game=gta5&search=configflags#ePedConfigFlags

Before you submit this PR, please make sure:

- You have read the contribution guidelines: Yes
- You include an example that validates your change: N/A
- Your English is grammatically correct: Yes

### CODE REVIEW

File: PED/SetPedConfigFlag.md [hunk 1]
```diff
@@ -477,3 +483,4 @@ enum ePedConfigFlags {
 * **flagId**: 
 * **value**: 
 
+The source for this enum is [here](https://alexguirre.github.io/rage-parser-dumps/dump.html?build=3095&game=gta5&search=ePedConfigFlags+#ePedConfigFlags).
```
Comment (PsychoShock) [inline PED/SetPedConfigFlag.md:486]: You can remove this. External links are not really appreciated in docs.

**Review by PsychoShock — CHANGES_REQUESTED** [review]
Hello,

I have left a personnal comment on this PR. Nice job. I still think a big enum like this should be moved to FiveM game docs instead since actual docs doesnt support big enums like this. I think a small work can be done to add a description on every flag if we moved there. I'll leave it to @4mmonium.

Thanks :)

**Review by alberttheprince — COMMENTED** [review]
Removed link as requested

**Comment by coalaura** [comment]
0x4376ABF2 = CPED_CONFIG_FLAG_DisableHealthRegenerationWhenStunned

**Comment by AvarianKnight** [comment]
> 0x4376ABF2 = CPED_CONFIG_FLAG_DisableHealthRegenerationWhenStunned

Going to leave this out for right now, if you want to update this in another pr feel free to do so

**Pull request merged**